### DISCUSSION TITLE
[neovim] Stop suggesting the code that only changes indent level

### DISCUSSION THREAD

**Discussion by creasty** [community/community#20736] [Copilot Conversations]
It is so annoying...
Please consider making the behavior configurable.

https://user-images.githubusercontent.com/1695538/179389082-5a1b5f75-e8f1-4bbe-88d5-314612721f03.mov

I've been using a fork with this patch:
https://github.com/creasty/copilot.vim/commit/2d3a3e8a0c0ae524eacb87e120c42cdcab6035e0

---

`|` represents the cursor position.

### 1. Initial state

```
    [
|   ];
```

### 2. Suggestion being displayed

```
    [
|   ];
```

### 3. After hitting tab

```
    [
    ];|
```

There's no way to get what I expected other than to use space keys for instance.

### 1. Initial state

```
    [
    |   ];
```

### 2. Suggestion being displayed

```
    [
   |];
```

Now you can't tell how many indents it actually has.

**Comment by tpope** [comment]
The latest release dials back the aggressiveness on lines like this.

> There's no way to get what I expected other than to use space keys for instance.

I would encourage you to try CTRL-T, a handy built-in which increases the indent regardless of cursor position.

**Reply by creasty** [reply]
> The latest release dials back the aggressiveness on lines like this.

That's good.

> I would encourage you to try CTRL-T, a handy built-in which increases the indent regardless of cursor position.

Yeah, I know CTRL-T does the trick.
However, the problem is actually a cognitive load of needing to decide which key to use.
